eclass/ros-catkin.eclass

```diff
@@ -163,16 +163,19 @@ ros-catkin_src_prepare() {
 	append-cxxflags '-std=c++11'
 }
 
+ROSPREFIX="/opt/ros/${ROSDISTRO}"
```

Comment:
We can't move everything like that: imagine you rebuild one package that expects its dependencies to be in /opt/ros while those have been built before this eclass change; package will fail to find its dependencies

Also, ROSDISTRO will be undefined until all ebuilds have been updated, and a global scope ROSPREFIX would prevent from having the same package built for 2 or more distros